src/main/java/io/spring/githubchangeloggenerator/ApplicationProperties.java

```diff
@@ -140,11 +143,25 @@ public static class Section {
 		 */
 		private final Set<String> labels;
 
-		public Section(String title, @DefaultValue("default") String group, IssueSort sort, Set<String> labels) {
+
+		/**
+		 * Whether to generate a link to each issue in the changelog.
+		 */
+		private final boolean generateLinks;
```

Comment:
Why do we need a second property for this? The `Issues` inner-class already has one.